splash/qtrender_lua.py

```diff
@@ -758,6 +758,36 @@ def callback(reply):
     def autoload_reset(self):
         self.tab.autoload_reset()
 
+    def _check_mouse_coordinantes(self, x, y):
+        try:
+            assert isinstance(x, int) or isinstance(x, float), "x"
+            assert isinstance(y, int) or isinstance(y, float), "y"
+        except AssertionError as e:
```

Comment:
A nitpick: this exception won't be fired if Splash is started with `python -O` or with `PYTHONOPTIMIZE` env variable set; it is not too important, but could you please rewrite it without assert?